Makefile

```diff
@@ -94,3 +95,11 @@ log:
 
 journal:
 	journalctl -b 0 -r --since "1 hour ago"
+
+test-shell: install
+	env GNOME_SHELL_SLOWDOWN_FACTOR=2 \
+		MUTTER_DEBUG_DUMMY_MODE_SPECS=1500x1000 \
+	  MUTTER_DEBUG_DUMMY_MONITOR_SCALES=1 \
+		dbus-run-session -- gnome-shell --nested --wayland
```

Comment:
Did not know this. Thanks - I suppose `make dev` can be updated to `make test-x` and this one `make test-wayland`?

Reply:
We can clean up Makefile in future.